--- dnsmasq-1.9/Makefile~	2002-10-30 00:49:54.000000000 +0100
+++ dnsmasq-1.9/Makefile	2002-10-30 00:51:30.000000000 +0100
@@ -1,6 +1,8 @@
 OBJS = cache.o rfc1035.o dhcp.o dnsmasq.o
 BINDIR = /usr/local/sbin
 MANDIR = /usr/local/man
+CC=gcc
+OPT=-02
 
 all : dnsmasq
 
@@ -8,11 +10,11 @@
 	rm -f *~ $(OBJS) dnsmasq
 
 .c.o: dnsmasq.h
-	gcc -O2 -Wall -W -c $*.c 
+	$(CC) $(OPT) -Wall -W -c $*.c 
 
 
 dnsmasq : $(OBJS) dnsmasq.h config.h
-	gcc -o $@  $(OBJS) $(LIBS)
+	$(CC) -o $@  $(OBJS) $(LIBS)
 
 install : dnsmasq
 	install -d $(BINDIR) -d $(MANDIR)
